src/ProjectNeon/Assets/Data/Templates/Target.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+/**
+ * A target for some effect in battlefield
+ */
+public abstract class Target : ScriptableObject {
+
+    [SerializeField] public List<Member> members;

```

Comment:
All Serialize field names must be `camelCased` .

Reply:
@SilasReinagel But it is camel cased!